Makefile

```diff
@@ -17,7 +17,12 @@ endif
 
 ifeq ($(detected_OS),Darwin)
  GOBIN_SHARED_LIB_EXT := dylib
- GOBIN_SHARED_LIB_CFLAGS := CGO_ENABLED=1 GOOS=darwin
+ # GOBIN_SHARED_LIB_CFLAGS := CGO_ENABLED=1 GOOS=darwin
+ # Temporary solution for cross-compilation on Mac M1
```

Comment:
This commen't doesn't actually explain __why__ we are doing this. The actual reason is:

>Qt14.x doesn't provides arm64 binaries for the platform

https://github.com/status-im/status-go/pull/3131#issuecomment-1406303234